src/templates/TemplateData.ts

```diff
@@ -37,6 +37,13 @@ export class TemplateData {
         'TimerTrigger'
     ];
 
+    private readonly _javaTemplates: string[] = [
+        'HttpTrigger',
```

Comment:
I can't get an HttpTrigger to build:
```
Erics-MacBook-Pro:aa ericjizba$ mvn clean package
[INFO] Scanning for projects...
[INFO]
[INFO] ------------------------------------------------------------------------
[INFO] Building Azure Java Functions 1.0-SNAPSHOT
[INFO] ------------------------------------------------------------------------
[INFO]
[INFO] --- maven-clean-plugin:2.5:clean (default-clean) @ aa ---
[INFO] Deleting /Users/ericjizba/TestRepos/aa/aa/target
[INFO]
[INFO] --- maven-resources-plugin:3.0.2:resources (default-resources) @ aa ---
[INFO] Using 'UTF-8' encoding to copy filtered resources.
[INFO] skip non existing resourceDirectory /Users/ericjizba/TestRepos/aa/aa/src/main/resources
[INFO]
[INFO] --- maven-compiler-plugin:3.1:compile (default-compile) @ aa ---
[INFO] Changes detected - recompiling the module!
[INFO] Compiling 2 source files to /Users/ericjizba/TestRepos/aa/aa/target/classes
[INFO] -------------------------------------------------------------
[WARNING] COMPILATION WARNING :
[INFO] -------------------------------------------------------------
[WARNING] /Users/ericjizba/TestRepos/aa/aa/src/main/java/com/function3/Function.java: Some input files use unchecked or unsafe operations.
[WARNING] /Users/ericjizba/TestRepos/aa/aa/src/main/java/com/function3/Function.java: Recompile with -Xlint:unchecked for details.
[INFO] 2 warnings
[INFO] -------------------------------------------------------------
[INFO] -------------------------------------------------------------
[ERROR] COMPILATION ERROR :
[INFO] -------------------------------------------------------------
[ERROR] /Users/ericjizba/TestRepos/aa/aa/src/main/java/com/function/HttpTriggerJava.java:[23,55] incompatible types: java.lang.Object cannot be converted to java.lang.String
[INFO] 1 error
[INFO] -------------------------------------------------------------
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 0.962 s
[INFO] Finished at: 2017-11-27T13:22:25-08:00
[INFO] Final Memory: 16M/298M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.1:compile (default-compile) on project aa: Compilation failure
[ERROR] /Users/ericjizba/TestRepos/aa/aa/src/main/java/com/function/HttpTriggerJava.java:[23,55] incompatible types: java.lang.Object cannot be converted to java.lang.String
[ERROR] -> [Help 1]
[ERROR]
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
```

Reply:
Yes recently the java function runtime has an update so the maven side need to do some stuff on this.

Issue #83